File: src/main/java/eterea/api/rest/model/Empresa.java
/**
 * 
 */
package eterea.api.rest.model;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table(name = "empresa")
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
public class Empresa extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -7878849276350845468L;

	@Id
	@Column(name = "emp_id")
	private Integer empresaId;

	@Column(name = "emp_neg_id")
	private Integer negocioId;

	@Column(name = "nombre")
	private String razonSocial;

	@Column(name = "emp_fantasia")
	private String nombreFantasia;

	@Column(name = "domicilio")
	private String domicilio;

	@Column(name = "telf")
	private String telefono;

	@Column(name = "cuit")
	private String cuit;

	@Column(name = "ingbrutos")
	private String ingresosBrutos;

	@Column(name = "nroestablecimiento")
	private String numeroEstablecimiento;

	@Column(name = "sedetimbrado")
	private String sedeTimbrado;

	@Column(name = "inicioactividades")
	private String inicioActividades;

	@Column(name = "condicioniva")
	private String condicionIva;

	@Column(name = "unegocio")
	private Byte unidadNegocio;

	@Column(name = "emp_diainicial")
	private Integer diaInicial;

	@Column(name = "emp_neg_id_comision")
	private Integer negocioIdComision;

	@Column(name = "emp_conectaunif")
	private Byte conectaUnificado;

	@Column(name = "certificado")
	private String certificado;

}

File: src/main/java/eterea/core/service/kotlin/model/Empresa.kt
package eterea.core.service.kotlin.model

import eterea.core.service.model.Auditable
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
data class Empresa(

    @Id @Column(name = "emp_id")
    var empresaId: Int? = null,

    @Column(name = "emp_neg_id")
    var negocioId: Int? = null,

    @Column(name = "nombre")
    var razonSocial: String? = null,

    @Column(name = "emp_fantasia")
    var nombreFantasia: String? = null,

    @Column(name = "domicilio")
    var domicilio: String? = null,

    @Column(name = "telf")
    var telefono: String? = null,

    @Column(name = "cuit")
    var cuit: String? = null,

    @Column(name = "ingbrutos")
    var ingresosBrutos: String? = null,

    @Column(name = "nroestablecimiento")
    var numeroEstablecimiento: String? = null,

    @Column(name = "sedetimbrado")
    var sedeTimbrado: String? = null,

    @Column(name = "inicioactividades")
    var inicioActividades: String? = null,

    @Column(name = "condicioniva")
    var condicionIva: String? = null,

    @Column(name = "unegocio")
    var unidadNegocio: Byte? = null,

    @Column(name = "emp_diainicial")
    var diaInicial: Int? = null,

    @Column(name = "emp_neg_id_comision")
    var negocioIdComision: Int? = null,

    @Column(name = "emp_conectaunif")
    var conectaUnificado: Byte? = null,

    @Column(name = "certificado")
    var certificado: String? = null

) : Auditable()
